pkg/sanity/controller.go

```diff
@@ -254,6 +253,154 @@ var _ = Describe("CreateVolume [Controller Server]", func() {
 			})
 		Expect(err).NotTo(HaveOccurred())
 	})
+	It("should not fail when requesting to create a volume with already exisiting name and same capacity.", func() {
```

Comment:
This looks good, but the only thing is that an error may be returned if the driver does not support `RequiredBytes`, like NFS. You will want to check the error returned to be either `OUT_OF_RANGE` or `UNIMPLEMENTED`.

Reply:
@lpabon thanks a lot for your review, I will address it.

Reply:
done